### ISSUE TITLE
MOve 3d Object

### ISSUE THREAD

**Issue by athiselvam** [eh3rrera/ARKitSceneKitExample#2]
How to move to this 3D object in real world?

**Comment by eh3rrera** [comment]
Hi, sorry I don't think I understand your question.

Once you put a 3D object in the scene, its position is fixed. But you can move it programmatically by changing either the [position](https://developer.apple.com/documentation/scenekit/scnnode/1408026-position) or [transformation](https://developer.apple.com/documentation/scenekit/scnnode/1407964-transform) attributes on the [SCNNode](https://developer.apple.com/documentation/scenekit/scnnode).

**Comment by athiselvam** [comment]
How do I move, rotate and scale based on touch action in a 3D object?

Can you alter this example to move the object in real time, and rotate the object using two fingers touch. Scale the 3D object's width and height.

**Comment by eh3rrera** [comment]
Sorry, I don't have time to modify this project. I'll consider making a tutorial about how to do that in the future.

I think the official Apple's demo has some code to handle one and two fingers gestures in ARKit: https://developer.apple.com/sample-code/wwdc/2017/PlacingObjects.zip

**Comment by Zanbrachrissik** [comment]
> Hi, sorry I don't think I understand your question.
> 
> Once you put a 3D object in the scene, its position is fixed. But you can move it programmatically by changing either the [position](https://developer.apple.com/documentation/scenekit/scnnode/1408026-position) or [transformation](https://developer.apple.com/documentation/scenekit/scnnode/1407964-transform) attributes on the [SCNNode](https://developer.apple.com/documentation/scenekit/scnnode).

Does this mean that if I move my camera towards the 3D object, it gets bigger? Or the distance between the camera and the 3D object is fixed?

**Comment by eh3rrera** [comment]
Yes, it gets bigger: https://www.youtube.com/watch?v=-o7qr1NpeNI
